README.md

```diff
@@ -44,7 +44,7 @@ Requires Node.js 6+ and npm 3+.
 #### Quick Start
 
 1. `git clone git@github.com:nteract/commuter.git`
```

Comment:
`Open http://localhost:3000` needs to be part of the 1) setup instruction since it doesn't apply to Heroku (it randomly assigns port)